### PR TITLE
Adding dotnet core runtime 2.2 and 3.0 jnlp-agents

### PR DESCRIPTION
**Pull request by StephenDryden**

### CODE REVIEW

File: coreruntime/22/Makefile [hunk 1]
```diff
@@ -0,0 +1 @@
+include ../common.mk
```
Comment (slide) [inline coreruntime/22/Makefile]: I think you want `../../common.mk`

File: coreruntime/30/Makefile [hunk 2]
```diff
@@ -0,0 +1 @@
+include ../common.mk
```
Comment (slide) [inline coreruntime/30/Makefile]: `include ../../common.mk`

**Comment by oleg-nenashev** [comment]
FTR https://github.com/jenkinsci/docker-jnlp-slave/issues/96

**Comment by StephenDryden** [comment]
I've made the suggested changes but must admit I'm not clear on the difference between agent.jar and slave.jar. Please can you clarify? I followed the examples in the other jnlp-agent docker files.

**Comment by slide** [comment]
The Jenkins project is trying to get rid of all used of the word slave. It's a process.

**Comment by StephenDryden** [comment]
Hi @rtyler - are you able to review + merge?

**Pull request merged**